staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -270,3 +270,8 @@ func (list ErrorList) Filter(fns ...utilerrors.Matcher) ErrorList {
 	// FilterOut takes an Aggregate and returns an Aggregate
 	return fromAggregate(err.(utilerrors.Aggregate))
 }
+
+// Is reports whether any Error in err's chain matches target.
+func (e *Error) Is(target error) bool {
+	return (string(e.Type) == target.Error())
```

Comment:
This is actually supposed to check that the `target` error is `field.Error` and then compare fields. https://blog.golang.org/go1.13-errors has a good example of `Is` implementation.